stdlib/3/collections/__init__.pyi

```diff
@@ -13,9 +13,11 @@ from typing import (
 # These are exported.
 # TODO reexport more.
 from typing import (
+    Container as Container,
```

Comment:
Good catch, but I'd like to see this (and anything else that's not strictly part of werkzeug) in a separate PR.